datalad_ria/tests/test_ria_basics.py

```diff
@@ -0,0 +1,137 @@
+from datalad.api import Dataset
+
+from datalad.customremotes.ria_utils import (
+    create_ds_in_store,
+    create_store,
+)
+from datalad.distributed.ora_remote import (
+    SSHRemoteIO,
+)
+from datalad.distributed.tests.ria_utils import (
+    common_init_opts,
+    populate_dataset,
+)
+from datalad.support.exceptions import (
+    CommandError,
+)
+from datalad.tests.utils_pytest import (
+    assert_in,
+    assert_not_in,
+    assert_raises,
+    assert_repo_status,
+    has_symlink_capability,
+)
```

Comment:
many of these (``assert_not_in``, ``assert_repo_status``, ``has_symlink_capability``) are not among those helpers datalad-next declares as "safe to import". Small test adjustments could be to

- change imports to datalad_next.tests.utils where possible
- change those helpers that can't get imported from datalad-next to something better in the test code

Reply:
`assert_in` should become `assert <something> in <something else>`

Same for `not_in`

`assert_raises` should become `with pytest.raises(<exception>):  ...`

Reply:
I replaced `assert_in`, `assert_not_in`, `assert_raises` in [3a78099](https://github.com/datalad/datalad-ria/pull/58/commits/3a78099584487764d9f13a98307f6f628e44aab2)